pkg/controller/risingwave_controller.go

```diff
@@ -426,13 +383,12 @@ func (c *RisingWaveController) SetupWithManager(mgr ctrl.Manager) error {
 		// Can't watch an optional CRD. It will cause a panic in manager.
 		// So do not uncomment the following line.
 		// Owns(&prometheusv1.ServiceMonitor{}).
-		Complete(c)
+		Complete(metrics.NewControllerMetricsRecorder(c, "RisingWaveController", gvk))
 }
 
 func NewRisingWaveController(client client.Client, recorder record.EventRecorder) *RisingWaveController {
 	return &RisingWaveController{
 		Client:   client,
 		Recorder: recorder,
-		name:     "RisingWaveController",
```

Comment:
IMHO we should keep this. By now we have two controllers: 

- RisingWave Controller 
- ScaleView Controller 

If we pass in the the names we may use this information later on for debugging or even to pass it on into the metrics as a parameter. That way we would know which  controller threw an error or which controller reconcile count was just increased. What do you think?